### PR TITLE
feature: new rule to check that region_id in station_information exists

### PR DESCRIPTION
**Pull request by hbruch**
This PR adds a new rule, which checks that `station.region_id` in `station_information` feed is defined in the `system_regions` feed.

### CODE REVIEW

File: src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    DocumentContext rawSchemaDocumentContext,
+    Map<String, JSONObject> feeds
+  ) {
+    JSONObject systemRegionsFeed = feeds.get("system_regions");
+    JSONObject regionIdSchema = rawSchemaDocumentContext.read(
+        REGION_IDS_SCHEMA_PATH
+    );
+
+    if (systemRegionsFeed != null) {
+      JSONArray vehicleTypeIds = JsonPath
```
Comment (testower) [inline src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java]: Probably a copy-paste error

```suggestion
      JSONArray regionIds = JsonPath
```

File: src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        .parse(systemRegionsFeed)
+        .read("$.data.regions[*].region_id");
+      regionIdSchema.put("enum", vehicleTypeIds);
```
Comment (testower) [inline src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java]:
```suggestion
      regionIdSchema.put("enum", regionIds);
```

**Review by testower — CHANGES_REQUESTED** [review]
Just a minor naming error. Also, can we add this new rule to the README please :)

File: src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java (hunk 1 above)
Comment (hbruch) [reply src/main/java/org/entur/gbfs/validation/validator/rules/NoInvalidReferenceToRegionInStationInformation.java]: Yes...

**Review by testower — APPROVED** [review]

**Pull request merged**